core/frontend/src/components/common/DevicePathHelper.vue

```diff
@@ -80,7 +80,7 @@ export default Vue.extend({
     height: {
       type: String,
       required: false,
-      default: '500px',
+      default: '250px',
```

Comment:
I think that a better solution is to use `em` over `px`, since `em` is relative to the text size. But I remember a discussion with @rafaellehmkuhl about that a time ago.

Reply:
browser pixels are not real pixels. also I use 100% scaling on my linux setup (where these prints were taken)